client/views/actions/ctcp.tpl

```diff
@@ -1,2 +1 @@
-<span role="button" class="user {{colorClass from}}" data-name="{{from}}">{{from}}</span>
-<b>{{ctcpType}}</b> {{{parse ctcpMessage}}}
+<span role="button" class="user {{colorClass from}}" data-name="{{from}}">{{from}}</span><b>{{ctcpType}}</b> {{{parse ctcpMessage}}}
```

Comment:
Not tested this one, but I assume it'll need a space between "from" and "ctcpType"